x/btcstaking/types/msg.go

```diff
@@ -145,6 +168,13 @@ func (m *MsgAddCovenantSigs) ValidateBasic() error {
 	if m.UnbondingTxSig == nil {
 		return fmt.Errorf("empty covenant signature")
 	}
+
+	_, err := m.UnbondingTxSig.ToBTCSig()
+
+	if err != nil {
```

Comment:
```suggestion
	if _, err := m.UnbondingTxSig.ToBTCSig(); err != nil {
```
IIRC this is the practice recommended by Go. Same comments for other places in this file